examples/FirebaseDemo_ESP8266/FirebaseDemo_ESP8266.ino

```diff
@@ -20,11 +20,19 @@
 #include <ESP8266WiFi.h>
 #include <FirebaseArduino.h>
 
-void setup() {
-  Serial.begin(9600);
+// Set these to run example.
+#define FIREBASE_HOST "example.firebaseio.com"
+#define FIREBASE_AUTH "token_or_secret"
+#define WIFI_SSID "SSID"
+#define WIFI_PASSWORD "PASSWORD"
+
+void ConnectWifi(const String& ssid, const String& password = "") {
```

Comment:
I also feel that user might want to have control over that part when modifying the sample:
- blink led while connecting
- display stuff on a sceren

So it seems to me it's friendler to leave it as part of the setup (user might be scared to modify it if it lives in a separate function).